lib/src/json_schema/json_schema.dart

```diff
@@ -2305,25 +2367,25 @@ class JsonSchema {
       });
 
   /// Validate, calculate and set the value of the 'maxProperties' JSON Schema keyword.
-  _setMaxProperties(dynamic value) => _maxProperties = TypeValidators.nonNegativeInt('maxProperties', value);
+  _setMaxProperties(Object value) => _maxProperties = TypeValidators.nonNegativeInt('maxProperties', value);
 
   /// Validate, calculate and set the value of the 'minProperties' JSON Schema keyword.
-  _setMinProperties(dynamic value) => _minProperties = TypeValidators.nonNegativeInt('minProperties', value);
+  _setMinProperties(Object value) => _minProperties = TypeValidators.nonNegativeInt('minProperties', value);
 
   /// Validate, calculate and set the value of the 'patternProperties' JSON Schema keyword.
-  _setPatternProperties(dynamic value) =>
+  _setPatternProperties(Object value) =>
```

Comment:
I see how this was necessary for your work :trollface:

Reply:
I took a 30 minute swipe at making the code null safe. I think making these things 'Object' could help with that effort later.